src/dwn/interfaces/records.js

```diff
@@ -0,0 +1,103 @@
+import { DwnConstant } from '@tbd54566975/dwn-sdk-js';
+
+import { Interface } from './interface.js';
+import { Record } from '../models/record.js';
+import { dataToBytes } from '../../utils.js';
+
+export class Records extends Interface {
+  constructor(dwn) {
+    super(dwn, 'Records');
+  }
+
+  get create() {
+    return this.write;
+  }
```

Comment:
i dont think this will propagate `this` correctly.  id suggest having it be this instead
```
async create(...args) {
  return this.write(...args);
}
```
that will also make it appear correctly in the Web Inspector (i.e. a function instead of a getter)

Reply:
Addressed in d08b75f64cfd5a8f21e0e613fecc4477e9bc78e8